libraries/AP_Scripting/docs/docs.lua

```diff
@@ -452,6 +452,82 @@ function motor_factor_table_ud:roll(index) end
 ---@param value number
 function motor_factor_table_ud:roll(index, value) end
 
+-- network socket class
+---@class SocketAPM_ud
+local SocketAPM_ud = {}
+
+-- desc
+function SocketAPM(param1) end
```

Comment:
You could rename this to `Socket` this "getter" is the only place the name is exposed to the script.

Reply:
done